src/client/keycode.h

```diff
@@ -49,6 +49,11 @@ class KeyPress
 		return !(*this == o);
 	}
 
+	// Used for e.g. std::unordered_set
```

Comment:
unordered sets don't require ordering, did you mean `set`?

Reply:
Yes; I think I originally write it for `std::hash<KeyPress>` and left it at the wrong place. Anyway I corrected it in https://github.com/luanti-org/luanti/pull/15933/commits/56314715e4578f883a552566ea1e1053bd189609.